Misc/NEWS.d/next/Core and Builtins/2019-12-21-14-18-32.bpo-39091.dOexgQ.rst

```diff
@@ -0,0 +1,2 @@
+Type check the result of constructing the new exception instance in
+:c:func:`_PyErr_CreateException`. Patch by coolreader18
```

Comment:
`_PyErr_CreateException()` is not part of the public C API, so I don't think `c:func` should be used here.